packages/react-charts/src/components/ChartTheme/ChartTheme.ts

```diff
@@ -1,8 +1,26 @@
 import { VictoryThemeDefinition } from 'victory-core';
 
-// Note: Victory incorrectly typed ThemeBaseProps.padding as number instead of PaddingProps
-export interface ChartThemeDefinitionInterface extends VictoryThemeDefinition {
-  skeleton?: {
+/**
+ * Chart component theme definition
+ * @private
+ * @beta
+ */
+export interface ChartComponentThemeDefinitionInterface {
+  axis?: VictoryThemeDefinition;
```

Comment:
It's unexpected to me that type of each of these props is now VictoryThemeDefinition.

I guess it's because the types spread all the same props in this [API doc](https://commerce.nearform.com/open-source/victory/docs/victory-theme).

So i guess that's not a comment that requires any action. just me documenting what I learned for maybe future me 😇

Reply:
They always were of type VictoryThemeDefinition, but wanted to be explicit.